src/app/shared/services/open-dialog/open-dialog.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing'
+
+import { OpenDialogService } from './open-dialog.service'
+
+describe('OpenDialogService', () => {
+  let service: OpenDialogService
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({})
+    service = TestBed.inject(OpenDialogService)
+  })
+
+  xit('should be created', () => {
```

Comment:
Sembrerebbe che i test falliscano a causa delle dependecy injection di material

Reply:
Andrebbe fixato in qualche maniera ;)